lean4-infoview-api/src/infoviewApi.ts

```diff
@@ -105,7 +105,7 @@ export interface InfoviewApi {
   /** Must fire with the server's initialization message when the server is started or restarted. */
   serverRestarted(serverInitializeResult: InitializeResult): Promise<void>;
   /** Must fire with the server's message when the server is stopped. */
-  serverStopped(serverStoppedResult: string): Promise<void>;
+  serverStopped(serverStoppedMsg: string, serverStoppedReason: string): Promise<void>;
```

Comment:
Is there a reason to split up the message here?  From what I can tell, the only difference is that reason is printed in red.  As far as I'm concerned, we can just print everything in red.  I'd rather keep it simple if possible.

*If* we really need to split it, then please introduce an interface:
```suggestion
  serverStopped(serverStoppedReason: ServerStoppedReason): Promise<void>;
  
...


export interface ServerStoppedReason {
  message: string;
  reason: string;
}
```

Reply:
Thanks for the feedback, yes it is just for clarity of the error reporting, a title of what we think went wrong and a reason from the actual exception that we have no idea what it says but it might be useful.

Can you educate us on why the InfoView Api can't take multiple parameters and needs a structured object instead?

Reply:
Oh it can take multiple fields, no problem.  It's just that `[string, string]` is a very uninformative type and it would be better to use descriptive fields names here.

Reply:
Got it, makes sense, Mariana, can you make that change?